card/mod/05_standard/spec/set/type/file_spec.rb

```diff
@@ -37,41 +40,59 @@
 
     describe 'view: source' do
       it 'renders url' do
-        expect(subject.format.render(:source)).to eq("/files/~#{subject.id}/#{subject.last_action_id}.txt")
+        expect(subject.format.render(:source)).to(
+          eq "/files/~#{subject.id}/#{subject.last_action_id}.txt"
+        )
       end
     end
 
-    context "updated file card" do
+    context 'updated file card' do
       before do
-        subject.update_attributes! file: File.new( File.join FIXTURES_PATH, 'file2.txt' )
+        subject.update_attributes! file: File.new(File.join FIXTURES_PATH, 'file2.txt')
```

Comment:
Line is too long. [87/80]<br>Add parentheses to nested method call `File.join FIXTURES_PATH, 'file2.txt'`.